chapters/substitution.adoc

```diff
@@ -2,21 +2,36 @@
 .Definition
 Robots are substituted by the <<Robot Handler, robot handler>> of the respective team. No other team member is allowed to take robots out or put robots in.
 
-. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
-.. Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
-.. Take robots out. If a robot that is taken out is closer than 1 meter to the intersection of the halfway line with one of the touch lines, another robot can be put in right away. Otherwise the robot handler has to wait until the next substitution opportunity.
+. The <<Robot Handler, robot handler>> should prefer to use long sleeves and colors that won't interfere with the vision system.
+
+. Robots can always be substituted during game play if the following specifications are respected:
+
+.. Is at least partially inside the <<Field Surface, field margin>>
+.. At a distance from the halfway line that must not exceed 30 centimeters
+.. The ball must be at least 2 meters away from the halfway line
+
+. If the previous specifications are respected, the <<Robot Handler, robot handler>> is allowed to perform the following operations.
```

Comment:
Originally, the rules were defined with the order and the maximum number of robots in mind, that's why there were those two options (put in, take out). The idea was: The robot handler can put robots in until the maximum number is reached. Then the robot handler can take robots out again. That way, a team had to play with one robot less until the next stoppage.

Now in the new context, this formulation does not make much sense to me anymore. I would just simplify it to one sentence.